api/blog.js

```diff
@@ -57,3 +58,61 @@ export async function fetchBlogPostPage(locale, slug) {
     throw error
   }
 }
+
+const blogPostsPerPage = 10
+
+export function fetchBlogPostsList(locale, page) {
+  const skip = blogPostsPerPage * (page - 1)
+  return fetchEntriesForContentType(
+    'blogPost',
+    {
+      unpackItems: false,
+      locale,
+      skip,
+      limit: blogPostsPerPage,
+      order: '-fields.date',
+      'fields.title[exists]': true,
+    },
+  )
+    .then((response) => {
+      const totalNumberOfBlogPosts = response.total
+      const totalNumberOfPages = Math.ceil(totalNumberOfBlogPosts / blogPostsPerPage)
+      const entries = response.items
+      const numberOfLastPostOnPage = skip + entries.length
+      const isLastPage = numberOfLastPostOnPage >= totalNumberOfBlogPosts
+
+      const blogPosts = entries.map(({ sys, fields }) => {
+        const author = fields.authorTeamMember
+          ? unwrapTeamMember(fields.authorTeamMember).name
+          : fields.authorExternal
+        const teaserImage = unwrapImage(fields.teaserImage, { q: jpegQuality, w: 1000 })
+        const date = formatDate(fields.date)
+        return ({
+          id: sys.id,
+          title: fields.title,
+          date,
+          author,
+          teaserText: fields.metaDescription,
+          teaserImage,
+          slug: fields.slug,
+        })
+      })
+
+      return ({
+        totalNumberOfPages,
+        isLastPage,
+        blogPosts,
+      })
+    })
+}
+
+export async function fetchBlogLandingPage(locale, page = 1) {
+  const pageContent = await fetchSingleEntry('pageBlogHome', locale)
+  const fetchBlogPostsResult = await fetchBlogPostsList(locale, page)
```

Comment:
You can still `await Promise.all([...])` to make the two requests run in parallel.